docs/content/envoy/workflows.en.md

```diff
@@ -0,0 +1,214 @@
+    autonumber
+    participant Oc as Originator User
+    participant O as Originator Envoy
+    participant B as Beneficiary Envoy
+    participant Bc as Beneficiary User
+
+    rect rgb(27, 206, 159 )
+        note over O,B: RPC 1
+        Oc ->> O: Transaction Draft
+        O ->> B: Started
+        B ->> O: Pending
+        O ->> Oc: Transaction Pending
+        B ->> Bc: Transaction Review
+    end
+
+    critical
+        Bc ->> Bc: KYC and Sanctions Checks
+    end
+
+    rect rgb(240,186,200)
+        note over O,B: RPC 2
+        B ->> O: Repair
+        O ->> B: Pending
+        B ->> Bc: Transaction Pending
+        O ->> Oc: Transaction Repair
+    end
+
+    critical
+        Oc ->> Oc: Required KYC and Repair
+    end
+
+    rect rgb(240,186,200)
+        note over O,B: RPC 3
+        O ->> B: Review
+        B ->> O: Pending
+        O ->> Oc: Transaction Pending
+        B ->> Bc: Transaction Review
+    end
+
+    critical
+        Bc->> Bc: Validate fixes meet requirements
+    end
+
+    rect rgb(27, 206, 159 )
+        note over O,B: RPC 4
+        Bc ->> B: Accept Transaction
+        B ->> O: Accepted
+        O ->> B: Accepted
+        B ->> Bc: Transaction Accepted
+        O ->> Oc: Transaction Accepted
+    end
+
+    critical
+        Oc ->> Oc: Perform on chain transaction
+    end
+
+    rect rgb(27, 206, 159 )
+        note over O,B: RPC 5
+        Oc ->> O: Complete Transaction
+        O ->> B: Completed
+        B ->> O: Completed
+        O ->> Oc: Transaction Completed
+        B ->> Bc: Transaction Completed
+    end
+```
+
+## Transaction Rejected and/or Canceled
+
+```mermaid
+sequenceDiagram
+    autonumber
+    participant Oc as Originator User
+    participant O as Originator Envoy
+    participant B as Beneficiary Envoy
+    participant Bc as Beneficiary User
+
+    rect rgb(27, 206, 159 )
+        note over O,B: RPC 1
+        Oc ->> O: Transaction Draft
+        O ->> B: Started
+        B ->> O: Pending
+        O ->> Oc: Transaction Pending
+        B ->> Bc: Transaction Review
+    end
+
+    critical
+        Bc ->> Bc: KYC and Sanctions Checks
+    end
+
+    rect rgb(240,186,200)
+        note over O,B: RPC 2
+        Bc ->> B: Reject Transaction (no retry/repair)
+        B ->> O: Reject
+        O ->> B: Reject
+        B ->> Bc: Transaction Rejected
+        O ->> Oc: Transaction Rejected
+    end
+
+    critical
+        Oc ->> Oc: Stop on chain transaction
+    end
+```
+
+## Transaction States and Actions
+
+A transaction may be in one of the following states:
+
+| Status | Description |
+|---|---|
+| Unspecified | The transfer state is unknown or purposefully not specified (zero value) |
+| Draft | The transaction has been created but no secure envelopes have been sent yet |
+| Pending | Awaiting a response from the counterparty |
+| Review | Requires review by a compliance officer and some action taken |
+| Repair | An error has been sent back from the counterparty requesting a change |
```

Comment:
In the UI, I noticed that if I reject a transaction and select request repair the `status` is `pending` and not `repair`.

Reply:
I updated https://app.shortcut.com/rotational/story/26770/status-bugs to include that fix.